app/views/step_by_step_pages/publish_or_delete.html.erb

```diff
@@ -57,7 +55,9 @@
           <% end %>
 
           <% if @step_by_step_page.can_discard_changes? %>
-            <p class="govuk-body govuk-!-margin-top-5">Or</p>
+            <% if @step_by_step_page.can_be_published? %>
```

Comment:
Why are we checking for this? As far as I know, we don't currently have any situation where you can discard changes but you can't publish 🤔

Reply:
You could have added a new step that doesn't have content, to a published step by step, and then want to discard any and all new changes.